.idea/.name

```diff
@@ -1 +1 @@
-randovania
```

Comment:
Please don't change these `.idea` files.

FYI, if you're using PyCharm it _should_ be possible to open the root folder as a project

Reply:
How in the world di that happen? I didn't even open that and it didn't show up when I typed "git status"